packages/map/src/components/UsaMap/components/TerritoriesSection.tsx

```diff
@@ -33,6 +38,9 @@ const TerritoriesSection: React.FC<TerritoriesSectionProps> = ({ territories, lo
       return a.props.label.localeCompare(b.props.label)
     })
 
+  const SVG_GAP = 9
```

Comment:
We have an open PR that creates `/helpers/constants.ts` if you want to create a todo for that in the comments or move these to that file that could help with organization during merges.

Example:
https://github.com/CDCgov/cdc-open-viz/pull/2031/files#diff-3f91e4da51d66243cc5975357d105a0cdfa68c360db88e9d859d4e757738c729R1